jqModal.js

```diff
@@ -356,6 +363,7 @@
 		 * (Function)  onShow       - User defined callback function fired when modal opened.
 		 * (Function)  onHide       - User defined callback function fired when modal closed.
 		 * (Function)  onLoad       - User defined callback function fired when ajax content loads.
+		 * (String)		 blockMsgClass- User defined exclusion parent class which are allowed to get focus even outside the active modal (like the jQueryUI datepicker)
```

Comment:
@dpesch I'd prefer not to have any side effects. lets default this to `null` and conditionally apply the $.target.parents match (https://github.com/briceburg/jqModal/pull/25/files#diff-63a527e6356eabc413f23b70fb806783R328)  if its not null.